ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```
@@ -442,6 +442,8 @@ presubmits:
       - args:
         - --config-path
         - ./core-services/prow/02_config/_config.yaml
+        - --supplemental-prow-config-dir
+        - core-services/prow/02_config
```

```suggestion
        - ./core-services/prow/02_config
```

To keep the consistency?

---

I had that initially and that makes checkconfig fail :/